data/data-pipeline/data_pipeline/etl/sources/us_army_fuds/etl.py

```diff
@@ -0,0 +1,106 @@
+from pathlib import Path
+import geopandas as gpd
+import pandas as pd
+
+from data_pipeline.etl.base import ExtractTransformLoad, ValidGeoLevel
+from data_pipeline.utils import get_module_logger, download_file_from_url
+from data_pipeline.etl.sources.geo_utils import add_tracts_for_geometries
+
+logger = get_module_logger(__name__)
+
+
+class USArmyFUDS(ExtractTransformLoad):
+    """The Formerly Used Defense Sites (FUDS)"""
+
+    NAME: str = "us_army_fuds"
+
+    ELIGIBLE_FUDS_COUNT_FIELD_NAME: str
+    INELIGIBLE_FUDS_COUNT_FIELD_NAME: str
+    ELIGIBLE_FUDS_BINARY_FIELD_NAME: str
+    GEO_LEVEL: ValidGeoLevel = ValidGeoLevel.CENSUS_TRACT
+
+    def __init__(self):
+        self.FILE_URL: str = (
```

Comment:
I'm late to the party here, but why isn't the URL in the YAML?

Reply:
That I can't answer. @esfoobar-usds ?

Reply:
@lucasmbrown-usds and I went back and forth about it and since it's not really a parameter you'll use in other places, we thought it belonged right and center on the ETL class itself.